src/Intermediary.tsx

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+import React, { useState, useEffect } from "react";
+import logo from "./assets/logo.png";
+import { IntermediaryClient } from "../clients/IntermediaryClient";
+export const Intermediary: React.FunctionComponent = () => {
+  const [ownerBalance, setOwnerBalance] = useState(0);
+  const [intermediaryBalance, setIntermediaryBalance] = useState(0);
+  const client = new IntermediaryClient({
```

Comment:
We will need to instantiate an IntermediartyCoordinator rather than a Client. The `IntermediartyClient` class is Irene's view into one specific channel. The Coordinator contains multiple IntermediartyClients and helpers to allow events in one channel to trigger events in another.